tsconfig.json

```diff
@@ -1,11 +1,39 @@
+/*
+  This is a "Solution Style" tsconfig.json file, and is used by editors and TypeScript’s language server to improve development experience.
+  It is not intended to be used to perform a compilation.
+
+  To learn more about this file see: https://angular.io/config/solution-tsconfig.
+*/
+
+/*
+This doesn't ork with our path mappings. See: https://github.com/microsoft/TypeScript/issues/39632
```

Comment:
I'm pretty sure it should be either work or, in some unexpected cases, orc :D

Reply:
:D